src/Adapter/AdapterInterface.php

```diff
@@ -36,3 +36,10 @@ public function getNormalizedIdentifier($model);
      */
     public function getUrlsafeIdentifier($model);
 }
+
+if (!interface_exists(\Sonata\CoreBundle\Model\Adapter\AdapterInterface::class)) {
```

Comment:
This is wrong. The autoloader won't find the `Sonata\CoreBundle\Model\Adapter\AdapterInterface` class, unless this file is loaded.

Reply:
If `Sonata\CoreBundle\Model\Adapter\AdapterInterface` has been autoloaded through the alias defined in my core bundle PR, this condition will return false, and the alias will not be defined. And it is what you want, because otherwise, you get a crash: https://3v4l.org/NqR2o